### PR TITLE
split and move HU

### PR DESCRIPTION
**Pull request by pvpurcarcosmin**
refs: https://github.com/metasfresh/metasfresh/issues/9926

### CODE REVIEW

File: backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+;
+
+-- 2020-09-11T07:14:17.275Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para_Trl (AD_Language,AD_Process_Para_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Process_Para_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Process_Para t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N') AND t.AD_Process_Para_ID=541858 AND NOT EXISTS (SELECT 1 FROM AD_Process_Para_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Process_Para_ID=t.AD_Process_Para_ID)
+;
+
+-- 2020-09-11T07:19:45.182Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541339,'S',TO_TIMESTAMP('2020-09-11 10:19:45','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','sysConfig.transform.enableMovement',TO_TIMESTAMP('2020-09-11 10:19:45','YYYY-MM-DD HH24:MI:SS'),100,'N')
```
Comment (metas-ts) [inline backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql:23]: * please add a short description
* the value `sysConfig.transform.enableMovement` is a bit uncommon..is there a reason we have to prefix it with `sysConfig`.`?

=> i suggest you orientate on the sysconfig `de.metas.ui.web.handlingunits.process.WEBUI_M_HU_Transform.AllowNewTUsWithInfiniteCapacity` (AD_SysConfig_ID=541082), minus any typos it contains

File: backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+;
+
+-- 2020-09-11T07:38:08.289Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,ColumnName,Created,CreatedBy,DisplayLogic,EntityType,FieldLength,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,578072,0,540763,541859,20,'ShowWarehouseID',TO_TIMESTAMP('2020-09-11 10:38:08','YYYY-MM-DD HH24:MI:SS'),100,'1=0','de.metas.ui.web',0,'Y','N','Y','N','N','N','Show warehouse ID',110,TO_TIMESTAMP('2020-09-11 10:38:08','YYYY-MM-DD HH24:MI:SS'),100)
```
Comment (metas-ts) [inline backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql:43]: allright; not displayed=>no trl and help text required

**Review by metas-ts — CHANGES_REQUESTED** [review]
i hope it's a quickie..

File: backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql (hunk 1 above)
Comment (pvpurcarcosmin) [reply backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql:23]: > the value sysConfig.transform.enableMovement is a bit uncommon..is there a reason we have to prefix it with sysConfig.`?

not really, just lack of inspiration I guess :)) adding a description now

Comment (pvpurcarcosmin) [reply backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql:23]: changed it anyway 👍

**Review by metas-ts — APPROVED** [review]

**Pull request merged**